runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MonitoringInfoConstants.java

```diff
@@ -41,7 +41,7 @@
     public static final String TOTAL_MSECS = extractUrn(MonitoringInfoSpecs.Enum.TOTAL_MSECS);
     public static final String USER_COUNTER_PREFIX =
         extractUrn(MonitoringInfoSpecs.Enum.USER_COUNTER);
-    public static final String USER_DISTRIBUTION_COUNTER_PREFIX =
+    public static final String USER_DISTRIBUTION_PREFIX =
```

Comment:
Better keep original name. Otherwise it is inconsistent with USER_COUNTER_PREFIX.